### DISCUSSION TITLE
GitHub send private repo info to their service provider who made it public

### DISCUSSION THREAD

**Discussion by gertsteyn** [community/community#21972] [New to GitHub]
<p>I got this email from GitHub today:</p>
<p>“We’re writing to let you know that between January 2021 and September 2021, the following information about your repository was inadvertently made publicly viewable after being sent to a third-party vendor as part of metadata analysis of GitHub Pages sites; the name of the private repository and the GitHub username with ownership of the repository. No repository content or other private data was exposed as part of this incident.”</p>
<p>They go on to give an explanation on how they sent the data to <a href="http://urlscan.io" rel="noopener nofollow ugc">urlscan.io</a> (luckily not source code) <strong>BUT</strong> the part that bothers me is the qualification they add to the last sentence:</p>
<p><em>No repository content or other private data was exposed</em> <em><strong>as part of this incident.</strong></em></p>
<p>Are there other incidents where they did do this?</p>
<p>Sadly I could not ask them directly because they sent the email from <a href="mailto:no-reply@github.com">no-reply@github.com</a>, but the email did direct me to the experts on the Community forum.</p>
<p>How serious was the “Jan 2021 to Sep 2021” incident and are there others we should be aware of?</p>

**Comment by queenofcorgis** [comment, marked as answer]
<p>Apologies for the delay <a class="mention" href="/gertsteyn">@gertsteyn</a>. You are more likely to get a speedier response when posting your question(s) in the applicable category. For questions on this incident, we recommend following <a href="https://github.com/contact?form%5Bsubject%5D=Re:Reference+GH-0008275-4369&amp;tags=GH-0008275-4369">direct link</a> included in that email which will take you to Support.</p>
